src/main/java/com/findmyflight/findmyflight/service/emailnotificationreceiver/EmailNotificationReceiver.java

```diff
@@ -0,0 +1,21 @@
+package com.findmyflight.findmyflight.service.emailnotificationreceiver;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class EmailNotificationReceiver {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String address;
```

Comment:
Do you sure about @NotEmpty ? I think it will be @NotBlank, because we have string value on our address. 

https://www.baeldung.com/java-bean-validation-not-null-empty-blank

Reply:
Yes, @notblank is even better :)